data-serving/data-service/src/controllers/case.ts

```diff
@@ -16,30 +17,43 @@ export const get = (req: Request, res: Response): void => {
  * Handles HTTP GET /cases.
  */
 export const list = (req: Request, res: Response): void => {
-    res.send('Triggered list cases.');
+    Case.find({}, (err, c) => {
```

Comment:
seems like this is using callbacks, any chance moongoose supports both callbacks and promises like the mongodb nodejs driver? If not, no big deal, just asking.

Reply:
FYI I did that in my PR it feels cleaner imo than callbacks

https://masteringjs.io/tutorials/mongoose/promise

Reply:
SG, updated.